djangoevents/domain.py

```diff
@@ -47,8 +71,43 @@ class BaseEntity(BaseAggregate):
 
     OBSOLETE! Interface kept for backward compatibility.
     """
+    @classmethod
+    def is_abstract_class(cls):
+        return super().is_abstract_class() or cls is BaseEntity
```

Comment:
Not sure I understand the problem. This thing should work well for each of the following:

```python
class BaseEntity # cls==BaseEntity => True                 OK

class A(BaseEntity) # super().is_abstract_class => False

@abstract
class B(A): # super().is_abstract_class => True            OK
```